projects/canopy/src/lib/skeleton/skeleton.stories.ts

```diff
@@ -0,0 +1,202 @@
+import { Component, OnInit } from '@angular/core';
+
+import { withKnobs } from '@storybook/addon-knobs';
+import { moduleMetadata } from '@storybook/angular';
+
+import { LgButtonModule } from '../button/button.module';
+import { LgCardModule } from '../card/card.module';
+import { LgGridModule } from '../grid/grid.module';
+import { LgHeadingModule } from '../heading/heading.module';
+import { LgPaddingModule } from '../spacing/padding/padding.module';
+import { LgDataPointModule } from './../data-point/data-point.module';
+import { LgSkeletonModule } from './skeleton.module';
+
+interface Data {
+  title?: string;
+  subTitle?: string;
+  content?: string;
+  datapoint?: {
+    label: string;
+    value: string;
+    prefix?: string;
+    suffix?: string;
+  };
+}
+
+@Component({
+  selector: 'async-skeleton-loading-card',
+  template: `
+    <lg-card>
+      <lg-card-header>
+        <lg-card-title headingLevel="4">
+          <span lgSkeleton lgSkeletonWidth="20">{{ data?.title }}</span>
```

Comment:
This is an example where the `lg-card-title` component projects the content into another child component, which won't work. So I have instead wrapped the content in a span and added the directive to that.

Reply:
Might be an ok limitation if it means we can start using this now, do you think it's worth any further investigation? another pair of eyes?

Reply:
I think it's ok, I had an initial attempt at making a product card in our app, and because of the ways its built with container components, I think the easiest implementation is possibly just replicating the card structure as a separate component and using this directive to add the loading states to get the desired effect.

Reply:
I did have a little look to get my head around it more. There might be a way to target the class on a child node, but for now I feel like it might be a nice minimal solution. If the limitation poses a wider problem we could invest more effort in it.